core/src/main/scripts/importer/validateData.py

```diff
@@ -1195,7 +1195,7 @@ class ClinicalValidator(Validator):
 
     REQUIRE_COLUMN_ORDER = False
     PROP_IS_PATIENT_ATTRIBUTE = None
-    NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]","[completed]","[null]", ""]
+    NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]","[completed]","[null]", "", "[unknown]", "unknown", "NA", "N/A"]
     ALLOW_BLANKS = True
```

Comment:
please add these scenarios to the corresponding pyunit test. See also https://github.com/cBioPortal/cbioportal/pull/1398/commits/ca49289fc1bedd78ffb6552fb44eb1e5f3e79136 for an example.